pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotTaskManager.java

```diff
@@ -356,7 +357,8 @@ public void registerTaskGenerator(PinotTaskGenerator taskGenerator) {
    * Helper method to schedule tasks (all task types) for the given tables that have the tasks enabled. Returns a map
    * from the task type to the task scheduled.
    */
-  private synchronized Map<String, String> scheduleTasks(List<String> tableNamesWithType, boolean isLeader) {
+  @VisibleForTesting
+  public synchronized Map<String, String> scheduleTasks(List<String> tableNamesWithType, boolean isLeader) {
```

Comment:
Why do we need to expose this API instead of using `scheduleTasks()`?

Reply:
For this test, I set up multiple tables for different test purposes. I want to schedule the tasks only for one table for each test. It would be complicated to check the results if we schedule tasks for all tables in a single test function.

Reply:
Sorry was referring to the `scheduleTasks(String tableNameWithType)` which schedule the task for a single table

Reply:
IC, thanks for pointing it out. Updated.